app/controllers/ncr/work_orders_controller.rb

```diff
@@ -29,7 +29,7 @@ def update
 
       super
 
-      if @model_changing && !@model_instance.emergency # skip approvals if emergency
+      unless skip_state_update?
```

Comment:
This feels like one of those cases where some apt comments would explain the "why" and the method can just be the "what".